app/app_main.c

```diff
@@ -952,7 +952,7 @@ static void enable_kdf (ACVP_CTX *ctx) {
     /*
      * Enable KDF-135
      */
-    rv = acvp_enable_kdf135_tls_cap(ctx, ACVP_KDF135_TLS, &app_kdf135_tls_handler);
```

Comment:
Are we removing the KDF code from the app in github at some point ?

Reply:
Ahh so I hadn't added any of the new kdf code, but I'll take out the ones that are in there from earlier. Give me a minute :)

Reply:
Actually I think it may be good to leave what we have for reference. I had already taken out our API calls.

Reply:
OK, got it. registration is good, handler implementation bad.